components/menus/QiskitOrgMenu.vue

```diff
@@ -90,6 +90,7 @@ export default class extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '~/assets/scss/mq.scss';
```

Comment:
Since we don't want to have to import the same scss code in every vue file, we should use: https://www.npmjs.com/package/@nuxtjs/style-resources to solve this problem. :)

Reply:
If you need help to integrate it, let me know.

Reply:
Very useful, let's do it in a different task qiskit/qiskit.org#309

Reply:
cool!